openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -698,7 +698,7 @@ The Credential Endpoint issues a Credential as approved by the End-User upon pre
 
 Communication with the Credential Endpoint MUST utilize TLS. 
 
-The Client can request issuance of a Credential of a certain type multiple times, e.g., to associate the Credential with different public keys/Decentralized Identifiers (DIDs) or to refresh a certain Credential.
+The Client can request issuance of a Credential of a certain type multiple times, e.g., to associate the Credential with different public keys/Decentralized Identifiers (DIDs) or to refresh a certain Credential. It is at the Credential Issuer's discretion how many copies of a Credential it is willing to issue, how often, and whether reissued Credentials automatically update if the underlying claims have changed.
```

Comment:
err, PR #293 here on [line 1399](https://github.com/openid/OpenID4VCI/pull/293/files#diff-1f424614b35a9899813079f1b1f6218631a2aedd993368ccb89bb81a9eda0289R1399) introduces batch_size, so there is a way for the issuer to communicate how many credentials it is willing to issue in a batch, so it is a little more than "at the discretion of the issuer".

also I am not sure what "automatically update" means. there is no way for the issuer to push into the wallet an updated credential, the wallet need to talk to the credential endpoint explicitly to get credentials refreshed.

Reply:
I would be ok with this

````
The Client can request the issuance of a Credential of a certain type multiple times, for example, to associate the Credential with different public keys or Decentralized Identifiers (DIDs), or to refresh a certain Credential. However, it is important to note that the ability to do so depends on whether the Issuer supports the multiple issuance of the same Credential Type or the refresh of the Credential.
````

Reply:
I would rather prefer this as a suggestion in PR#293 to avoid the merge conflict.

Reply:
yes we can do this, see: https://github.com/openid/OpenID4VCI/pull/293#discussion_r1632079784

Reply:
Is this PR still relevant?